androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/home/HomeDrawer.kt

```diff
@@ -0,0 +1,97 @@
+package co.nimblehq.avishek.phong.kmmic.android.ui.screen.home
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.Divider
+import androidx.compose.material.MaterialTheme.typography
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color.Companion.White
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import co.nimblehq.avishek.phong.kmmic.android.R
+import co.nimblehq.avishek.phong.kmmic.android.ui.theme.ApplicationTheme
+import co.nimblehq.avishek.phong.kmmic.android.ui.theme.Nero90
+import coil.compose.AsyncImage
+
+@Composable
+fun HomeDrawer(
+    userUiModel: UserUiModel?,
+    appVersion: String,
+    onLogoutClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier
+            .statusBarsPadding()
+            .fillMaxSize()
+            .background(Nero90)
+            .padding(horizontal = 20.dp)
+    ) {
+        Spacer(modifier = Modifier.height(80.dp))
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalArrangement = Arrangement.SpaceBetween,
+            verticalAlignment = Alignment.CenterVertically
+        ) {
```

Comment:
I think you are not aware of we can set `padding` at the `Modifier` instead of using `Spacer` 😬 .
```suggestion
        Row(
            modifier = Modifier
                .fillMaxWidth()
                .padding(top = 80.dp),
            horizontalArrangement = Arrangement.SpaceBetween,
            verticalAlignment = Alignment.CenterVertically
        ) {
```

Reply:
Oops, I mixed up `padding` and `paddingFromBaseline` — thought `padding` was measured from the baseline, so used `Spacer` since I found it easier to match the spacings with Figma. Going to replace them with `padding`s. 🙏🏽